static/src/stylesheets/head.content.scss

```diff
@@ -43,3 +43,4 @@
 @import 'module/atoms/_interactive';
 @import 'module/atoms/_youtube';
 @import 'module/atoms/_media';
+@import 'module/atoms/_storyquiz';
```

Comment:
`.content.scss` would be the place to be for this file. :) I'd go this way and only move in into `head.` if there are problems with it ...